dev/quickinstance/main.tf

```diff
@@ -0,0 +1,70 @@
+# module "base" {
```

Comment:
## Arc IaC Automation generated
Next steps:
 - Please fill these parameters, parameter definition can be found [here](https://github.com/Infrastructure-as-code-Automation/ALL-quickstart-template-terraform/blob/main/README.md)
 - After fill and commit, resolve this comment and merge the pull request 

If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com